lib/ex_doc/formatter/html.ex

```diff
@@ -143,9 +144,9 @@ defmodule ExDoc.Formatter.HTML do
       |> Enum.map(&Task.await(&1, :infinity))
 
     api_reference_headers =
-      if(Enum.empty?(modules),    do: [], else: [{"Modules", "modules"}]) ++
-      if(Enum.empty?(exceptions), do: [], else: [{"Exceptions", "exceptions"}]) ++
-      if(Enum.empty?(protocols),  do: [], else: [{"Protocols", "protocols"}])
+      if Enum.empty?(modules),    do: [], else: [{"Modules", "modules"}] ++
```

Comment:
This is definitely wrong. It changes the precedence of the code. If credo is asking us to remove the parenthesis on `if`, then it is definitely wrong. Maybe if we are using do/end blocks, but not for one-line ifs. /cc @rrrene